update upload command README.md (#2110)
demisto/demisto-sdk: demisto_sdk/commands/upload/README.md

    if true will skip all upload packs validations, use just when migrate existing custom content entities to custom content packs to override all the entities with the packs.

* **-x, --xsiam**

>     if true will upload the pack to XSIAM server. Must be used together with -z

Reviewer: ```suggestion
    uploads the pack to a XSIAM server. Must be used together with -z
```
